ofl/baloobhaijaan/DESCRIPTION.en_us.html

```diff
@@ -14,7 +14,7 @@
 <p><ul>
 <li><a href="https://fonts.google.com/specimen/Baloo">Baloo</a> for Devanagari</li>
 <li><a href="https://fonts.google.com/specimen/Baloo+Bhai">Baloo Bhai</a> for Gujarati</li>
-<li>Baloo Bhaijaan for Urdu (Nastaʿlīq Arabic)</li>
+<li>Baloo Bhaijaan for Arabic (Urdu)</li>
```

Comment:
IMO, there is nothing Urdu specific in this design (unlike, say, Noto Nastaliq Urdu which uses the Urdu style of Nastaliq), so I’d drop the Urdu part since the font will support more than Urdu in the next update.